src/scenario/data.h

```diff
@@ -81,10 +81,35 @@ enum {
     ALLOWED_BUILDING_MONUMENTS = 48,
 };
 
-struct win_criteria_t {
+#define MAX_WIN_CRITERIA 10
+
+typedef enum {
+    WIN_CRITERIA_NONE,
+    WIN_CRITERIA_POPULATION_MINIMUM,
+    WIN_CRITERIA_CULTURE,
+    WIN_CRITERIA_PROSPERITY,
+    WIN_CRITERIA_PEACE,
+    WIN_CRITERIA_FAVOR,
+    WIN_CRITERIA_SURVIVAL_YEARS,
+    WIN_CRITERIA_TIME_LIMIT,
+
+    // TODO: Try to implement these
+    WIN_CRITERIA_BUILD_X_BUILDINGS, // Build at least a quantity of a specific building type
+    WIN_CRITERIA_EVOLVE_X_HOUSES,   // Build at least X houses of the given level or higher
+    WIN_CRITERIA_EARN_X_DENARII,    // Have at least X denarii in city coffers
+    WIN_CRITERIA_SELL_X_GOODS,      // Sell at least X of the specific trade good to any trade partner
+    WIN_CRITERIA_HOLD_ROMAN_GAMES,  // Hold roman games at coluseum at least once (requested by Fairbuy)
+    WIN_CRITERIA_GIFT_X_DENARII,    // Spend at least X denarii on gifts to caesar
+
+    WIN_CRITERIA_END
```

Comment:
Maybe goals related to the gods?
WIN_CRITERIA_GODS_GRAND_TEMPLE_COUNT
WIN_CRITERIA_GODS_AVERAGE_HAPPINESS
WIN_CRITERIA_GODS_TOTAL_BLESSING_RECEIVED

Not sure. Just ideas.

Reply:
Once we get this framework in place, we can add any new criteria we want. These ones here are definitely good ideas (assuming we keep track of how many blessings have been received, which I don't know if that's recorded in the .sav)